### PR TITLE
Ming/animation

### PR DESCRIPTION
**Pull request by mingbeam**
### How would the reviewers verify the solutions
- make pictures in the info section animate when scrolling
- create useFadeIn hook for animation
- create sales channel, payment, social commerce animation (render the components individually)
- trigger animations while scrolling in the info section using Waypoint

https://user-images.githubusercontent.com/90749495/156735883-98cc7eb9-cc0f-4818-9bfe-a8d879ddcd99.mp4

### Preflight checklist

- [ ] **Manual testing** has been performed according to the acceptance criteria.
- [ ] _(heavy style change)_ The solution has been tested on the major browser (Chrome, Firefox, Safari, Microsoft Edge).

### CODE REVIEW

File: components/landing_page/InfoSection.module.css [hunk 1]
```diff
@@ -1,26 +1,16 @@
 .wrapper {
   text-align: center;
```
Comment (oukunanan) [inline components/landing_page/InfoSection.module.css:2]: Do we need this?

File: components/landing_page/animation/SocialCommerceAnimation.module.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  top: 0px;
+  left: 0px;
+}
+
+.animation5_2 {
+  position: absolute;
+  width: 53px;
+  top: 19px;
+  left: 127px;
+  z-index: 1;
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/SocialCommerceAnimation.module.css]: all removed krub

File: components/landing_page/animation/SocialCommerceAnimation.module.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  height: 100%;
+  margin-left: 20px;
+}
+
+.animation5_1 {
+  position: absolute;
+  width: 524px;
+  top: 0px;
+  left: 0px;
+}
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/SocialCommerceAnimation.module.css]: removed krub

File: components/landing_page/animation/SocialCommerceAnimation.module.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
+.container {
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/SocialCommerceAnimation.module.css:2]: when I removed it, it looked like this ah krub
Screenshot (1664)

File: components/landing_page/animation/SocialCommerceAnimation.module.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  width: 100%;
+  height: 100%;
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/SocialCommerceAnimation.module.css]: removed krub

File: components/landing_page/animation/PaymentAnimation.module.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
+.container {
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/PaymentAnimation.module.css:2]: when i removed it, the margin (changed to top, left) does not work ah krub

File: components/landing_page/animation/PaymentAnimation.module.css [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  width: 100%;
+  height: 100%;
+  margin: 20px 0 0 40px;
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/PaymentAnimation.module.css]: changed margin to top, left krub

File: components/landing_page/animation/SalesChannelAnimation.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+  return (
+    <div className={styles.container}>
+      <animated.div className={styles.sales_channel_1} style={fadeIn1}>
+        <Image src={sales_channel_1} alt="Sales Channel 1" priority />
+      </animated.div>
+
+      <animated.div className={styles.sales_channel_2} style={fadeIn2}>
+        <Image src={sales_channel_2} alt="Sales Channel 2" priority />
+      </animated.div>
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/SalesChannelAnimation.tsx]: swapped and removed z-index from css krub

File: components/landing_page/InfoSection.module.css (hunk 1 above)
Comment (mingbeam) [reply components/landing_page/InfoSection.module.css]: removed krub

File: components/landing_page/animation/SocialCommerceAnimation.module.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (oukunanan) [reply components/landing_page/animation/SocialCommerceAnimation.module.css:2]: Hmm, can you elarborate which part it changes na kub? I didn't notice the changes.

**Review by oukunanan — APPROVED** [review]
Approved in advance.

File: components/landing_page/animation/SocialCommerceAnimation.module.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/animation/SocialCommerceAnimation.module.css:2]: the icons at the top shift to the left krub
here's the original version krub
image

File: components/landing_page/InfoSection.module.css [hunk 9]
```diff
@@ -54,6 +55,25 @@
   margin-top: -320px;
 }
 
+.message_animation_wrapper {
+  display: flex;
```
[the remark this answers is not in the document]
Comment (mingbeam) [reply components/landing_page/InfoSection.module.css:59]: removed krub

**Pull request merged**